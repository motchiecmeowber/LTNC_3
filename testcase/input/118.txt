INSERT x number
BEGIN
INSERT y string
INSERT z number
ASSIGN z 2300
ASSIGN x 12
ASSIGN x z
RPRINT
END
PRINT