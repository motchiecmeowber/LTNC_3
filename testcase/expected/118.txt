success
success
success
success
success
success
z//1 y//1 x//0
x//0